### PR TITLE
[New Feature]Support udf  when loading data

### PR DESCRIPTION
**Pull request by xinghuayu007**
## Proposed changes

Many time, our users want to use UDFs they developed to ETL the data when loading the data into Doris.But currently, `broker load` does not support to use UDF. As UDF belongs to a database, it needs to check whether it has the `SELECT` permission of the database. This patch try to solve this problem. 

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [x] If this change need a document change, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — CHANGES_REQUESTED** [review]
If FE restart, the user info will be lost.
So you may need to pesist the user info.
 
And the compilation has failed.

File: fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BulkLoadJob.java [hunk 1]
```diff
@@ -324,7 +329,7 @@ public void readFields(DataInput in) throws IOException {
         // The origin stmt does not be analyzed in here.
         // The reason is that it will thrown MetaNotFoundException when the tableId could not be found by tableName.
         // The origin stmt will be analyzed after the replay is completed.
-
+        userInfo = UserIdentity.read(in);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BulkLoadJob.java:332]: Modify the `read()` function, you have to update the meta version in `FeMetaVersion.java`

Comment (xinghuayu007) [reply fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BulkLoadJob.java]: > Modify the `read()` function, you have to update the meta version in `FeMetaVersion.java`

done

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**